frontend/tests/email/maildev-mailbox.ts

```diff
@@ -14,9 +14,10 @@ export class MaildevMailbox extends Mailbox {
     super(email);
   }
 
-  async fetchEmails(subject: EmailSubjects): Promise<Email[]> {
+  async fetchEmails(subject: EmailSubjects, extraText?: string): Promise<Email[]> {
```

Comment:
rather than `extraText` which is not a very descriptive name. Just make subject `EmailSubjects | string` that change should bubble up all the way to the tests